Formula/guetzli.rb

```diff
@@ -0,0 +1,26 @@
+class Guetzli < Formula
+  desc "Perceptual JPEG encoder"
+  homepage "https://github.com/google/guetzli"
+
+  url "https://github.com/google/guetzli/archive/v1.0.tar.gz"
+  sha256 "9766353d4bcfb9ea1c4018770d23a321986cdf1544d0805d14361686d7592c92"
+
+  head "https://github.com/google/guetzli.git"
+
+  depends_on "libpng"
+  depends_on "gflags"
+
+  resource "test_image" do
+    url "https://github.com/google/guetzli/releases/download/v1.0/bees.png"
+    sha256 "2c1784bf4efb90c57f00a3ab4898ac8ec4784c60d7a0f70d2ba2c00af910520b"
+  end
+
+  def install
+    system "make"
+    bin.install "bin/Release/guetzli"
```

Comment:
No `install` target, no man page, no library.
Just the command line reference implementation of [Google's new open-source JPEG encoder, Guetzli](https://research.googleblog.com/2017/03/announcing-guetzli-new-open-source-jpeg.html).
The repository also contains an undocumented [python script](https://github.com/google/guetzli/blob/master/tools/guetzli-compare.py) as well as a copy of the [butteraugli tool](https://github.com/google/butteraugli), which seem to have been used to create the comparison results for their paper. However, it seemed to me that neither are appropriate to be included in this formula.